CIP-XXXX/README.md

```diff
@@ -0,0 +1,46 @@
+---
+CIP: ?
+Title: CIP-30 ext: Account public key
+Category: Wallets
+Status: Proposed
+Authors:
+    - Ola Ahlman <ola.ahlman@tastenkunst.io>
+    - Andrew Westberg <andrewwestberg@gmail.com>
+Implementors: [Eternl, newm-chain]
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
+Created: 2023-09-03
+License: CC-BY-4.0
+---
+
+## Abstract
+This CIP extends [CIP-30 (Cardano dApp-Wallet Web Bridge)](https://cips.cardano.org/cips/cip30/) to provide an additional endpoint for dApp to get the extended account public key from a connected wallet.
+
+## Motivation: why is this CIP necessary?
+Normally it's up to the wallet to handle the logic for utxo selection, derived addresses etc through the established CIP-30 api. Sometimes however, dApp needs greater control due to subpar utxo selection or other specific needs that can only be handled by chain lookup from derived address(es). This moves the control and complexity from wallet to dApp for those dApps that prefer this setup. A dApp has better control and can make a more uniform user experience. By exporting only the account public key, this gives read-only access to the dApp.
+
+## Specification
```

Comment:
Is there a need to support wallet's proving ownership of the shared account key?

Perhaps CIP-30's signed data just be used (if the dApp derives payment/stake keys)?

Reply:
This would add extra friction. All we really need here is a popup asking for permission from the user.

Reply:
I think its enough to handle it through the normal enable() call, if this extension is requested, just show enable popup to either allow or disallow.

Reply:
Ah no, what I meant was adding a separate endpoint where the wallet signs with this key to prove ownership.
But I dont think this is needed as existing signData could be used (albeit with extra steps for the verifier).